### ISSUE TITLE
Make possible switch across different networks (changing config) from gui

### ISSUE THREAD

**Issue by Fantu** [netbirdio/netbird#2233]
Hi, thanks for improvement of gui settings.
I think it would be useful to also add the opportunity to switch across different networks (changing config) from gui.
I need to switch network often and actually I need to stop netbird, rename 2 config file in C:\programdata\netbird\ (on windows) and restart netbird to switch the network.
I often also have to check first with `netbird status -d` to understand which network I am initially connected to so as not to make a mistake in renaming the config also including the name of the network.
I suppose I'm not the only one who needs this, in my case because I do technical assistance for multiple customers and only a few have a "shared" netbird network (and separated internally by acl).

**Comment by rqi14** [comment]
This will be very helpful

**Comment by oskardotglobal** [comment]
This is a duplicate of #446

**Comment by Fantu** [comment]
No, #446 is for multiple network at once, this only for make easier/fast switch network

**Comment by oskardotglobal** [comment]
Sorry, then I seem to have misremembered what #446 does. Anyways, I might pick up this issue and write up a PR for it.

**Comment by mgarces** [comment]
Hi there, we still don't have this feature; in the meantime, can you test this simple [script](https://gist.github.com/mlsmaycon/88c3284a39d65cc055a1493f40a030cc) that allows you to switch between installed NetBird service installs. You can create a service using
```
sudo netbird service install --service <profile_name> --config /etc/netbird/<profile_name>.json
```
The script handles multiple `profiles` and also handles the default name for service named `netbird` which uses the default config file, `config.json`. 

Since you can only be running one service at a time, I ended up creating this script to handle the start/stop of each service and to show me which running "profile" is activated. I know it's a temporary solution, but will help you handle this scenario a bit better until we implement the feature.

**Comment by PizzaLovingNerd** [comment]
Closing this, as this should be implemented in to NetBird Profiles. Cheers!
